_includes/layouts/base.njk

```diff
@@ -5,11 +5,26 @@
 
 	<body>
 
-		<header>
-			<em>
-				<a href="{{ '/' | url }}">{{ metadata.title }}</a>
-			</em>
-			{% include "components/nav.njk" %}
+		<header class="header-main">
+			<div class="container container--wide container--flex">
+				<div class="header-main__logo">
+					<a href="{{ '/' | url }}">
+						<img src="/static/img/nlc-full-logoX600.png" alt="{{ metadata.title }}" />
+					</a>
+				</div>
+				<div class="header-main__nav">
+					{% include "components/nav.njk" %}
+				</div>
+				<div class="header-main__menu js-closed">
+					<div class="header-main__menu__text js-closed-text">Menu</div>
```

Comment:
We probably need to use screenreader specific text here to give some more context, I think that "Open menu" and "Close menu" would probably be enough.

Reply:
Makes sense from an accessibility point of view for sure. Changed.